src/main/python/pybuilder/plugins/python/pylint_plugin.py

```diff
@@ -44,4 +45,63 @@ def execute_pylint(project, logger):
     logger.info("Executing pylint on project sources")
 
     command_and_arguments = ["pylint"] + project.get_property("pylint_options")
-    execute_tool_on_modules(project, "pylint", command_and_arguments, True)
+    pylint_output_file_path = execute_tool_on_modules(project, "pylint", command_and_arguments, True)[1]
+
+    with open(pylint_output_file_path, 'r') as f:
+        file_content = f.read().splitlines()
+
+    module_name = ''
+    pylint_score = 0
+    pylint_change = 0
+    errors = 0
+    errors_info = ''
+    warnings = 0
+    warnings_info = ''
+    show_info_messages = project.get_property('pylint_show_info_messages')
```

Comment:
Do we really need separated properties `pylint_show_info_messages` and `pylint_show_warning_messages`? Could we use global project property `project.get_property('verbose')` instead?

Reply:
With verbose you can not decide if you want to see only warnings, moreover with verbose you will see a lot of other logs. I think it is good to have separate configuration for this.
What do you think about this @AlexeySanko ?

Reply:
Hi, @maciejpolanczyk 
I used idea that pylint shows all issues anyway. And `verbose` mode supposes that output could be big. :grin:

Reply:
pylint shows all issues in report file. I would like to have possibility to show only warnings or only infos from pylint on console as output from pybuilder without big output which will be created by 'verbose' from other plugins and pybuiilder. It will be easy to read them and I will not have to go to report file to find out what went wrong.
What do you think about it @AlexeySanko ? Or should we abandon this idea because it is not consistent with unit tests execution?

Reply:
On the other hand - cause of report parsing You can manage different type of issues. Why not?
Only one notice - `logger.info` is the highest level of logger and always shown. I think that for conventions/refactorings should be used `logger.debug`

Reply:
Do you mean all logger.info in this file or only the one which shows infos from pylint?
We decided for logger.info base on other plugins which use this level for printing information which should be print in regular run (without verbose)

Reply:
Moved to particular row - https://github.com/pybuilder/pybuilder/pull/512/files#r148052635